### PR TITLE
[fix](serde)fix the bug in DataTypeNullableSerDe.deserialize_column_from_fixed_json

### PR DESCRIPTION
**Pull request by hubgeter**
## Proposed changes
fix a bug in DataTypeNullableSerDe.deserialize_column_from_fixed_json.

The expected behavior of the `deserialize_column_from_fixed_json` function is to `insert` n values ​​into the column.

 However, when the `DataTypeNullableSerDe` class implements this function, the null_map column is `resize` to n, which does not insert n values ​​into it. Since this function is only used by the `_fill_partition_columns` of the `parquet/orc reader` and is not called repeatedly for a `get_next_block`, this bug is covered up.
before pr : #37377

### CODE REVIEW

**Comment by hubgeter** [comment]
run buildall

**Comment by hubgeter** [comment]
run buildall

**Review by morningman — DISMISSED** [review]
LGTM

File: be/src/vec/data_types/serde/data_type_nullable_serde.cpp [hunk 1]
```diff
@@ -138,8 +138,10 @@ Status DataTypeNullableSerDe::deserialize_column_from_fixed_json(
     auto& null_map = col.get_null_map_data();
     auto& nested_column = col.get_nested_column();
 
-    null_map.resize_fill(
-            rows, null_map.back()); // data_type_nullable::insert_column_last_value_multiple_times()
+    uint8_t val = null_map.back();
+    size_t new_sz = null_map.size() + rows - 1;
+    null_map.resize_fill(new_sz,
```
[the remark this answers is not in the document]
Comment (hubgeter) [reply be/src/vec/data_types/serde/data_type_nullable_serde.cpp:143]: good idea ! i will do it.

**Comment by hubgeter** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by kaka11chen — APPROVED** [review]
LGTM

**Pull request merged**